packages/zoe/src/contracts/callSpread.js

```diff
@@ -133,17 +135,24 @@ const start = zcf => {
     return floorDivide(multiply(PERCENT_BASE, numerator), denominator);
   }
 
-  function payoffOptions(price) {
-    // either offer might be exercised late, so we pay the two seats separately.
+  function payoffOptions(priceQuoteAmount) {
+    const { Price: price } = quoteMath.getValue(priceQuoteAmount)[0];
     const longShare = calculateLongShare(price);
+    // either offer might be exercised late, so we pay the two seats separately.
     reallocateToSeat(Position.LONG, longShare);
     reallocateToSeat(Position.SHORT, inverse(longShare));
   }
 
   function schedulePayoffs() {
-    terms.priceAuthority
-      .priceAtTime(terms.expiration, terms.underlyingAmount)
-      .then(price => payoffOptions(price));
+    E(terms.priceAuthority)
+      .priceAtTime(
+        terms.timer,
+        terms.expiration,
+        terms.underlyingAmount,
+        strikeBrand,
+      )
+      .then(quoteIssuer.getAmountOf)
```

Comment:
```suggestion
    quoteIssuer.getAmountOf(E(terms.priceAuthority)
      .priceAtTime(
        terms.timer,
        terms.expiration,
        terms.underlyingAmount,
        strikeBrand,
      ))
```

Reply:
Can we use the `quoteAmount` part of the `{ quoteAmount, quotePayment }` that we discussed this morning so we don't have to do `getAmountOf`?

Reply:
And also then we don't need to know about `quoteIssuer` so we can drop that